pipelines/common/src/main/java/com/google/fhir/analytics/model/DatabaseConfiguration.java

```diff
@@ -29,6 +29,7 @@
 
 @Getter
 @Setter
+@SuppressWarnings("NullAway")
```

Comment:
Why do we have this annotation at class level here but in the controller module you had it repeated for (all?) fields of some classes?

Reply:
This has has since been updated - https://github.com/google/fhir-data-pipes/pull/1482